src/__tests__/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/downloads.test.tsx

```diff
@@ -5,23 +5,24 @@ import userEvent from "@testing-library/user-event";
 import { computeAccessibleDescription } from "dom-accessibility-api";
 import React from "react";
 
+import removeLegacySlugSuffix from "../../../../../../../../../utils/slugModifiers/removeLegacySlugSuffix";
```

Comment:
don't introduce new relative paths. Use aliases

Reply:
Will do, sometimes hard to spot when it auto imports